gaia-icons.css

```diff
@@ -23,3 +23,20 @@
 	font-size: 30px;
 	-webkit-font-smoothing: antialiased;
 }
+
+[data-icon]:not([data-l10n-id]):not([aria-hidden="true"]):before {
+	/* Highlight cases where icons are not localized and still visible to screen
+	   reader */
+	background: rgba(255, 0, 0, 0.4);
+}
+
+[data-icon]:not([data-l10n-id]):not([aria-hidden="true"]):hover:after {
```

Comment:
Yeah hover's not super helpful on touch devices. The only reason I did not include the warning in the above style was because it would make the elements pretty broken. Otherwise, no objections to that.